tests/ui/parser/raw/raw-literal-keywords.stderr

```diff
@@ -9,12 +9,22 @@ error: expected one of `!`, `.`, `::`, `;`, `?`, `{`, `}`, or an operator, found
    |
 LL |     r#struct Test;
    |              ^^^^ expected one of 8 possible tokens
+   |
+help: you might have meant to write a field access
+   |
+LL |     r#struct.Test;
```

Comment:
Should we limit only suggest field access for downcase leading character?

Reply:
Good idea. Done.